spec/data/record_loader/barcode_printer_types/two_entry_example.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
All your yml files are called 'two_entry_example' which doesn't functionally matter but they should probably be renamed to something more relevant